umad-02-keys-and-authentication.md

```diff
@@ -25,42 +25,56 @@ section.
 
 ## Public Key Exchange
 
-VASPs expose their public keys to other VASPs by responding to `GET` requests at the endpoint
+TThe UMA protocol relies on X.509 certificates for public key exchange among VASPs. In order to obtain an X.509
+certificate, the VASP is required to submit a request to a trusted certificate authority (CA), which issues the
```

Comment:
Actually this step *isn't required*. The VASP ID authority is an added security/trust measure that's not in place quite yet. We don't have any VASP ID authorities onboarded today, so we can't enforce that everyone registers with one (nor should we). They'll just provide an added level of trust for VASPs if needed. For example:

I'm VASP2 and I get an incoming payment from someone I don't recognize, VASP1. Today, VASPs need to make a choice here: either they just trust VASP1 (after some review, etc) or they just decide they're not going to trust any VASP they haven't seen before and whitelisted ahead of time. Because the existing pool of UMA VASPs is small today, most VASPs are going with the latter. To be long-term sustainable, we need another way for VASPs to know whether they can trust a counterparty they don't know. When we have trusted VASP ID Authorities, VASPs can check if an ID authority that they trust has signed the certificate, and if so, trust the underlying VASP.

This is just an incremental improvement to the trust model, but can't be a full requirement for now (or possibly ever)